views/components/GraphTable/index.jsx

```diff
@@ -62,7 +46,7 @@ const GraphTable = ({
       <Table
         cellRenderer={cellRenderer}
         handleRequestSort={handleRequestSort}
-        data={visibleRows}
+        data={visibleRows()}
```

Comment:
why are we passing this in as a function? Can't we pass the data directly?

Reply:
In the snippet you shared with me this is a function, if we don't call the function then we don't have the data. I can call the function at the parent level if that's what you want.
`  const visibleRows = () => {
    if (!graph) return []
    return graph.graphTable.tableRows
      .slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
      .sort(comparator)
  }`

Reply:
yes, we should be passing down the data we need directly, so let's execute the function at the parent level.

Reply:
[ad92374](https://github.com/AMP-SCZ/dpdash/pull/695/commits/ad92374536d663d20d5658f70a43fc1bd0826682)